### ISSUE TITLE
MISSING_VAR false positive cfquery loops

### ISSUE THREAD

**Issue by TheRealAgentK** [cflint/CFLint#284]
Column names from previous (correctly scoped/vared) queries inside of a query loop will be reported as incorrectly vared with the MISSING_VAR rule. In the example below, variables.siteDetailList[siteID] = ... will trigger it and report on siteID.

```
<cfquery name="something" datasource="#variables.DSN#">
      SELECT
      	..., siteID, ...
      FROM
        ...
</cfquery>

<cfloop query="something">
    <cfscript>
        variables.siteDetailList[siteID] = ...
    </cfscript>
</cfloop>
```

**Comment by ryaneberly** [comment]
Good catch.  Thanks @TheRealAgentK

**Comment by TheRealAgentK** [comment]
Working fine now, thx!
